.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: Run tests and linting
```

Comment:
Many thanks for working on this.

Looking forward to see the new release out :)

One very minor suggestion. Consider using a very short name.

it will make it easier to check the status from the main GitHub Repo page.

This is a GitHub UI issue... but I think that a short name can help check the status for the the main branch

See how it looks for a project with many checks with a snort name.
With long names you only see "Run tests and linting`

![image](https://user-images.githubusercontent.com/204609/131343347-ede1d8d4-589b-4258-878c-c12f170e0961.png)